hydromt/metadata_resolvers/resolver_plugin.py

```diff
@@ -0,0 +1,5 @@
+"""Resolvers detected."""
+from .convention_resolver import ConventionResolver
+from .metadata_resolver import MetaDataResolver
+
+RESOLVERS: dict[str, MetaDataResolver] = {"convention_resolver": ConventionResolver()}
```

Comment:
Again, I think an enum of frozen dict would be better here if possible.

Reply:
I added some comments